### ISSUE TITLE
Replace another service stt and tts

### ISSUE THREAD

**Issue by lionnet1981** [ankitjayswal87/VoiceBot#1]
Is it okay if I try using different speech-to-text services like Vosk and a text-to-speech service like Coqui TTS? I've attempted to incorporate them into the project several times, but they haven't been successful. However, when I ran your code and the log asterisk: AGI Script speech-to-text.py completed, returning 0 but it is not transfer intent to rasa. Even after trying my own code, the outcome remained the same. If you have a moment, could you please assist me with this? Thank you.

**Comment by ankitjayswal87** [comment]
Yes, you can use different STT and TTS service but need to add function in the files speech_to_text.py and text_to_speech_cpass.py , Also in .env file need to modify flags TTS_ENGINE , RASA_HOST and RASA_PORT

**Comment by lionnet1981** [comment]
Launched AGI Script /var/lib/asterisk/agi-bin/speech_to_text.py
  == End MixMonitor Recording PJSIP/200-00000006
 speech_to_text.py,/tmp/1708706968.6-1: Entering into Python AGI...
 speech_to_text.py,/tmp/1708706968.6-1: Recording FileName is /tmp/1708706968.6-1.wav
 speech_to_text.py,/tmp/1708706968.6-1: The Detected Data: I want to book an appointment booking appointment
    -- <PJSIP/200-00000006>AGI Script speech_to_text.py completed, returning 0
    -- Executing [talk@outgoing:5] NoOp("PJSIP/200-00000006", "The Intent :  EntityName :  EntityValue : ") in new stack
with log above I successfully with my stt code but it can not query to rasa let the bot ask name the caller so you can give me some advice about install rasa with spacy

**Comment by lionnet1981** [comment]
You use rasa version 3.3.3 ,right! But I want to downgrade it to rasa 2.8.14 because I want to use rasax but I want to ask you about spacyNLP that you use in rasa 3.3.3 so when I did above , it is still working with your code because I see another code from rasa if we use query any information from rasa that must be use custom action. In your code, you did not use custom action, right
